libavformat/movenc.c

```diff
@@ -6125,6 +6125,15 @@ static int mov_init(AVFormatContext *s)
         return AVERROR(EINVAL);
     }
 
+    /* Call io_open call back if the flag is set (output format is "mp4", and using mp4 muxer) */
+    if (s->call_io_open && s->io_open) {
```

Comment:
So here it will be if  ( ! s->pb && s->io_open ) ...

Reply:
move_init() function (in movenc.c) is called from different paths in libavformat. I wanted to become sure it is the only path that we call io_open and not other execution paths.